catroid/src/main/java/org/catrobat/catroid/stage/ShowTextActor.java

```diff
@@ -124,6 +122,58 @@ private boolean isNumberAndInteger(String variableValue) {
 		}
 	}
 
+	private void drawText(Batch batch, String text, float posX, float posY) {
+		// Convert to bitmap
+		Paint paint = new Paint();
+		paint.setTextSize(15 * scale);
+		paint.setColor(android.graphics.Color.BLACK);
+		paint.setAntiAlias(true);
+		paint.setTextAlign(Paint.Align.LEFT);
+		float baseline = -paint.ascent();
+		int width = (int) (paint.measureText(text) + 0.5f);
+		int height = (int) (baseline + paint.descent() + 0.5f);
+		Bitmap bitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888);
+		Canvas canvas = new Canvas(bitmap);
+		canvas.drawText(text, 0, baseline, paint);
+		// Convert to texture
+		Texture tex = new Texture(bitmap.getWidth(), bitmap.getHeight(),
+				Pixmap.Format.RGBA8888);
+		GLES20.glBindTexture(GLES20.GL_TEXTURE_2D,
+				tex.getTextureObjectHandle());
+		GLUtils.texImage2D(GLES20.GL_TEXTURE_2D, 0, bitmap, 0);
+		GLES20.glBindTexture(GLES20.GL_TEXTURE_2D, 0);
+		bitmap.recycle();
+		// Draw and dispose
+		batch.draw(tex, posX, posY);
+		batch.flush();
+		tex.dispose();
+	}
+
+	private String convertToEnglishDigits(String value) {
```

Comment:
please write a unit test that checks this.

then we could even think about not skipping on the ui test, since if the string here is correct its going to be fine. and also, if the string "1023456789" is correctly translated here in this function, but something else breaks, the ui test would not recognize this. 
so eg. "1023456789" is not correctly rendered out to the stage, its not done correctly no matter what language it was coming from, and since EN is compared to other locales, incorrect is compared to incorrect...

Reply:
done, also I rewrote the test as OCR test, 
have a look at https://github.com/tesseract-ocr/tesseract